_posts/2022-02-25-newpipe-0.22.1.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: post
+title: "NewPipe 0.22.1 released: Fixes and adjustments"
+short: "NewPipe 0.22.1 released"
+date: 2022-02-25 22:00:00 + 01:00
+categories: [pinned, release]
+author: <a href="https://github.com/Stypox">@Stypox</a>
+image: newpipe
+excerpt_separator: <!-- more -->
+---
+
+Why is there a new release only six days after the previous one? Well, don't expect anything big… just a few adjustments to the previous version and some long-awaited bugfixes for lists! That's what you would expect from a point release, according to [semantic versioning](https://semver.org/), but we have never been that good at choosing release numbers, sorry about that.
+
+<!-- more -->
+
+### Fixed
+
+- Lists on big screen might finally work as expected! Now if the items loaded initially do not fill the whole screen, NewPipe will load another chunk of them, instead of showing a loading indicator **forever**. This long-awaited fix was brought to you by none other than the hero from last release, [@litetex](https://github.com/litetex). In the same pull request he also fixed some recurring crashes when scrolling through lists, caused by a misuse of the nasty Android `RecyclerView`. 
```

Comment:
```suggestion
- Lists on big screen might finally work as expected! Now if the initially loaded items do not fill the whole screen, NewPipe will load another chunk of them, instead of showing a loading indicator **forever**. This long-awaited fix was brought to you by none other than our hero from last release, [@litetex](https://github.com/litetex). In the same pull request he also fixed some recurring crashes when scrolling through lists, caused by a misuse of the nasty Android `RecyclerView`. 
```